File: 2.0/java/contacts/findContactLists.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.contacts.model.ContactList;
import com.callfire.api.client.api.contacts.model.request.FindContactListsRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        FindContactListsRequest request = FindContactListsRequest.create()
            .name("my list")
            .offset(0L)
            .limit(10L)
            .fields("items(id,name,size)")
            .build();
        Page<ContactList> contactLists = client.contactListsApi().find(request);
    }
}

File: 2.0/csharp/contacts/findContactLists.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Contacts.Model;
using CallfireApiClient.Api.Contacts.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request = new FindContactListsRequest
        {
            Name = "my list",
            Offset = 0,
            Limit = 10,
            Fields = "items(id,name,size)"
        };
        Page<ContactList> contactLists = client.ContactListsApi.Find(request);
    }
}
